README.md

```diff
@@ -132,9 +132,10 @@ We test all combinations of:
 
 | Instance  | Host OS & Kernel  | Guest Rootfs | Guest Kernel |
 |:----------|:------------------|:-------------|:-------------|
-| m5d.metal | al2    linux_4.14 | ubuntu 22.04 | linux_4.14   |
-| m6i.metal | al2    linux_5.10 |              | linux_5.10   |
-| m6a.metal | al2023 linux_6.1  |              |              |
+| c5n.metal | al2    linux_4.14 | ubuntu 22.04 | linux_4.14   |
+| m5n.metal | al2    linux_5.10 | ubuntu 22.04 | linux_5.10   |
```

Comment:
nit: there are duplicate entries for guest rootfs

Reply:
ugh, yes. I will fix.